BasicSample/Assets/Shared/Scripts/AppRemotingSample.cs

```diff
@@ -48,15 +50,60 @@ private void Awake()
                 // into an XR experience and it's too late to connect remoting.
                 if (xrDisplaySubsystem.running)
                 {
-                    SetObjectActive(gameObject, false);
-                    SetObjectActive(immersiveUI, false);
-                    SetObjectActive(flatUI, false);
+                    DisableConnection2DUI();
                     return;
```

Comment:
I think I found a new bug in this change; after opening and closing any feature scene, the "Disconnect Remoting" button on the main menu isn't visible.

If I had to guess, I'd expect the logic around these lines could be to blame, disabling some of the UI as the scene is re-entered, but then I'd expect we'd have seen this bug before.

Reply:
As we discussed, this bug was already existing. We can track it separately and still move these changes forward here.

Reply:
Yeah, The disconnect button in XR scenes will somehow disappear after switching scenes.  We can track this as a separate task.